setup.py

```diff
@@ -27,6 +28,32 @@ def load_module_from_path(module_name, path):
 ROOT_DIR = os.path.dirname(__file__)
 logger = logging.getLogger(__name__)
 
+
+def embed_commit_hash():
+    try:
+        commit_id = subprocess.check_output(
+            ['git', 'rev-parse', '--short', 'HEAD'], encoding="utf-8").strip()
```

Comment:
should we just grab the complete hash? this would give us the option of how to display it later.

Reply:
Seems reasonable; updated.